packages/bot-web-ui/src/components/dashboard/joyride-config.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+
+type TJoyrideConfig = Record<'showProgress' | 'showSkipButton' | 'spotlightClicks' | 'disableBeacon', boolean>;
+
+type TStep = {
+    label: string | boolean;
+    content: string;
+};
+
+const Step = ({ label, content }: TStep) => (
+    <div>
+        <h1>{label}</h1>
```

Comment:
We can use <Text> component instead. Just pass the `h` as the prop

Reply:
done ✅